### PR TITLE
add imagepullsecret option to goldilocks charts

### PR DESCRIPTION
**Pull request by elinar27**
**Why This PR?**
add option to deploy with private registry

**Checklist:**

* [x] I have updated the chart version in `Chart.yaml` following Semantic Versioning.
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] Any new values have been added to the README for the Chart, or helm-docs has been run for the charts that support it.

### CODE REVIEW

File: stable/goldilocks/values.yaml [hunk 1]
```diff
@@ -21,6 +21,8 @@ image:
   # image.pullPolicy -- imagePullPolicy - Highly recommended to leave this as `Always`
   pullPolicy: Always
 
+imagePullSecrets: []
```
Comment (sudermanjr) [inline stable/goldilocks/values.yaml:25]:
```suggestion
# imagePullSecrets -- A list of image pull secret names to use
imagePullSecrets: []
```

**Review by sudermanjr — CHANGES_REQUESTED** [review]
Overall looks great! Thanks for the contribution.

We use helm-docs to maintain the table, so I think this suggestion will help

**Review by sudermanjr — CHANGES_REQUESTED** [review]
Can you please run helm-docs with the sort flag documented here:

 `https://github.com/FairwindsOps/charts/blob/master/CONTRIBUTING.md#helm-docs`

**Review by sudermanjr — CHANGES_REQUESTED** [review]
Great! Only thing left is the CLA. Please see the comment from the CLA assistant and follow the directions there.

**Review by sudermanjr — APPROVED** [review]
LGTM. Thank you!

**Comment by elinar27** [comment]
Thanks to you :)

**Pull request merged**